taproot.html

```diff
@@ -202,18 +204,19 @@
 dd {
 	font-weight: bold;
 	text-decoration: underline;
+	margin: 0;
+	cursor: pointer;
 }
 dt {
 	margin-bottom: 1em;
 	text-align: justify;
 }
-
 </style>
 </head>
 <body>
 
 <div id="menu">
-<img src="taproot.svg" id="logo"/>
+<img src="taproot.svg" id="logo" width="250" />
```

Comment:
This will distort the aspect ratio of the image since we set height to 100% in CSS.

Reply:
In what browser are you seeing this? It looks fine in both Firefox and Chrome for me (see screenshot).

Reply:
I'm not seeing it, but HTML is not a fixed size format. 100% of the height may very well be 500px, for example.